lib/tmuxinator/hooks.rb

```diff
@@ -0,0 +1,39 @@
+module Tmuxinator
+  module Hooks
+    def hook_on_reattach
```

Comment:
Perhaps a helper method could/should be made to reduce the duplication in this module? There could be a `supported_hooks` array which the helper could be mapped over. This would also make adding new hooks much simpler.

I realize that's potentially YAGNI, but I was just reading the tmux documentation and there are _many_ more hooks users might want to make use of:

> Each tmux command has a before hook and an after hook and there are a number of hooks not associated with commands.